### PR TITLE
makepasswd: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

I could not find any homepage for this command.

### CODE REVIEW

File: pages/common/makepasswd.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# makepasswd
+
+> Generate and encrypt passwords.
+
+- Generate a random password:
```
Comment (schneiderl) [inline pages/common/makepasswd.md]: What are the defaults? I think it would be nice to state it (if it's not too long)>

Comment (schneiderl) [reply pages/common/makepasswd.md]: I'm asking that because I don't think it is truly random, it must follow a pattern.

Comment (Rudloff) [reply pages/common/makepasswd.md]: Oh right! I added a note about that.

Comment (schneiderl) [reply pages/common/makepasswd.md]: Thank you 😄 looks great

File: pages/common/makepasswd.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# makepasswd
+
+> Generate and encrypt passwords.
+
```
Comment (mebeim) [inline pages/common/makepasswd.md]:
```suggestion
> More information: <https://manpages.debian.org/stretch/makepasswd/makepasswd.1.en.html>

```
I think we can link the Debian man page here.

**Pull request merged**